tests/spec/fs.chmod.spec.js

```diff
@@ -39,6 +39,20 @@ describe('fs.chmod, fs.fchmod', function() {
     });
   });
 
+  it('should be a valid mode value to set permissions', function(done){
+    var fs = util.fs();
+
+    fs.mkdir('/file', function(err) {
```

Comment:
This `mkdir` call isn't necessary.  You can just use `/` as your directory (always exists).